site/en/blog/autoplay/index.md

```diff
@@ -0,0 +1,290 @@
+go with a review. Users interacted with the domain to get to the blog, so
+autoplay is allowed. However, the blog needs to explicitly delegate that
+privilege to the iframe in order for the content to autoplay.
+
+### Chrome enterprise policies
+
+It is possible to change the autoplay behavior with Chrome enterprise policies
+for use cases such as kiosks or unattended systems. Check out the [Policy
+List](policy-list) help page to learn how to set the autoplay related enterprise
+policies:
+
+- The `[AutoplayAllowed](autoplayallowed)` policy controls whether autoplay is allowed or not.
+- The `[AutoplayAllowlist](autoplayallowlist)` policy allows you to specify an allowlist of URL
+  patterns where autoplay will always be enabled.
+
+## Best practices for web developers
+
+### Audio/Video elements
+
+Here's the one thing to remember: Don't ever assume a video will play, and
+don't show a pause button when the video is not actually playing. It is so
+important that I'm going to write it one more time below for those who simply
+skim through that post.
+
+{% Aside 'gotchas' %}Don't assume a video will play, and don't show a pause
+button when the video is not actually playing.{% endAside %}
+
+You should always look at the [Promise](promise) returned by the play function
+to see if it was [rejected]:
+
+```js
+var promise = document.querySelector('video').play();
+
+if (promise !== undefined) {
+  promise.then(_ => {
+    // Autoplay started!
+  }).catch(error => {
+    // Autoplay was prevented.
+    // Show a "Play" button so that user can start playback.
+  });
+}
+```
+
+{% Aside 'caution' %}Don't play interstitial ads without showing any media
+controls as they may not autoplay and users will have no way of starting
+playback.{% endAside %}
+
+One cool way to engage users is to use muted autoplay and let them chose
+to unmute. (See the example below.) Some websites already do this effectively,
+including Facebook, Instagram, Twitter, and YouTube.
+
+```html
+<video id="video" muted autoplay>
+<button id="unmuteButton"></button>
+
+<script>
+  unmuteButton.addEventListener('click', function() {
+    video.muted = false;
+  });
+</script>
+```
+
+### Web Audio
+
+The [Web Audio API](web-audio) has been covered by autoplay since Chrome 71.
+There are a few things to know about it. First, it is good practice to wait for
+a user interaction before starting audio playback so that users is aware of
+something happening. Think of a "play" button or "on/off" switch for instance.
+You can also add an "unmute" button depending on the flow of the app.
+
+{% Aside %}If an `AudioContext` is created before the document receives a user
+gesture, it will be created in the "suspended" state, and you will need to call
+`resume()` after the user gesture.{% endAside %}
+
+If you create your `AudioContext` on page load, you'll have to call `resume()`
+at some time after the user interacted with the page (e.g., after a user clickes
```

Comment:
clickes? I believe it should be clicks or clicked no?

Reply:
Fixed.